src/components/content/content-blocks/figure/figure.tsx

```diff
@@ -23,6 +23,9 @@ export const Figure = ({
 			key={node.key}
 			{...attributes}
 		>
+			{node.elementId && (
+				<a id={node.elementId}></a>
```

Comment:
Aesthetic preference here, but: Wouldn't it make more sense to name it `anchor`? This is obviously a `named anchor`.

Reply:
I prefer to leave it `id`, like html switched from `<a name=...` to `a id=...`. Just a way to address elements.